boxEngine.py

```diff
@@ -0,0 +1,182 @@
+import os
+import random
+import time
+import platform
+import sys
+from configparser import ConfigParser
+from rich.console import Console
+
+console = Console()
+config = ConfigParser(comment_prefixes="#", delimiters="=")
+config.read("data/config/main.ini")
+
+# Color codes:
+def warn(stri):
+    console.print(f"[bold][bright_yellow]█ Warning! -> {stri}[/]")
+def error(stri):
+    console.print(f"[bold][bright_red]█ Error! -> {stri}[/]")
+def success(stri):
+    console.print(f"[bold][bright_green]█ Success! -> {stri}[/]")
+
+try:
+    configCheck1 = str(config["DEBUG"]["disableRem"])
+    if configCheck1 == "false":
+        print("# Reminder!! Every command is CaSe SeNsItIvE")
+    else: ...
+
+except:
+    exit("Error! Config file or an element is missing!")
+
+# cmd items not worth putting there
+cmdList = ["shellBasic","source","pyclockInit","passgenInit","help","quit","exit","vars","cvar","box"]
+ver = str(config["MAIN"]["version"])
+console.print(f"[ [bold][bright_green]OK[/] ][italic] boxEngine is loaded![/]")
+
+class boxEngine:
+
+    def box(string):
+        from git import Repo
+        repo = Repo()     
+
+    def rok():
+        return "Exit status:", 0
+    
+    def rno():
+        return "Something wrong happened, exit status:",0
+
+    def rmdir(path: str):
+        os.rmdir(path)
+
+    def mkdir(name: str):
+        os.mkdir(name)
+
+    def spamClear():
+        for i in range(100):
+            print("")
+
+    def cvar(): # this one wont write.
+        with open('data/txt/cvar.txt') as file:
+            print(f"cvar value = {file.read()}")
+            lines = file.readlines()
+
+
+        cvarI = input("cvar> ")
+        if cvarI == "exit": ...
+        elif cvarI == "change":
+            print("Input new string or integer to change.")
+            ce = input("cvar>change> ")
+            with open("data/txt/cvar.txt", "wt") as file:
+                for line in lines:
+                    file.write(ce)
+                print(f"New cvar value is:{ce}")
+
+    def vars():
+        config.read("data/config/vars.ini")
+        try:
+            readVars = str(config.items("VARS"))
+            print("Current vars are:")
+            print(readVars)
+        except:
+            boxEngine.error.data404()
+    
+    class error():
+        def noCvar():
+            error("File 'cvar.txt' in /data/txt/ wasn't found.")
+
+        def NotImplement():
+            warn("Sorry, but this function is not implemented yet!")
+
+        def ConfigFatal(Err):
+            console.print_exception(f"[bold][bright_red][Error!] Config file or an element is missing! -> {Err}")
+        
+        def data404():
+            error("Data needed to access wasn't found.")
+
+        def userFail():
+            error("Input Password failed too many times.")
+
+        def user404():
+            error("User not found.")
+
+    class funct:
+        def echo(x):
```

Comment:
Print exists, you don't  need an extra function to print (;

Reply:
> Print exists, you don't  need an extra function to print (;

Print in CLI? Its a function to be ran in bps cli.